frontend/src/lib/stores/userStore.ts

```diff
@@ -86,4 +88,6 @@ async function hash(input: string): string {
 	return hash;
 }
 
-export { createDummyUser, hash, users, UserStore, type UserData, type UserList };
+const currentUser = writable(null as unknown as UserRead);
```

Comment:
why `unknown`? isn't it either null or UserRead? i.e.g `null as null | UserRead`?

Reply:
hm, I was following error tips of the editor at this point, because conversion to UserRead gets flagged as an error and they get overwhelming at some point... I agree that it's unintuitive...